src/test/java/perosnal/spotifymatcher/service/UserServiceTest.java

```diff
@@ -0,0 +1,131 @@
+package perosnal.spotifymatcher.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import perosnal.spotifymatcher.model.User;
+import perosnal.spotifymatcher.repository.UserRepository;
+import perosnal.spotifymatcher.util.AuthorizedActionResult;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.lenient;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceTest {
+
+    @Mock
+    private UserRepository userRepository;
+    @Mock
+    private SpotifyApiService spotifyApiService;
+    @InjectMocks
+    private UserService userService;
+
+    @BeforeEach
+     void before() {
+        lenient().when(spotifyApiService.getIdByToken("")).thenReturn("");
+        lenient().when(userRepository.getById("")).thenReturn(new User());
+    }
```

Comment:
Please do not mock interactions in `@BeforeEach` method as this will affect all test cases.
[Here](https://stackoverflow.com/a/11837973/5922757) is an awesome StackOverflow answer regarding this.

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)